src/PythonClient/src/quixstreams/dataframes/models/serializers/base.py

```diff
@@ -73,6 +73,19 @@ class Serializer(abc.ABC):
     A base class for all Serializers
     """
 
+    @property
+    def extra_headers(self) -> MessageHeadersMapping:
```

Comment:
I don't see a huge issue with this approach overall, but maybe the language should better match the intention?

I like "default_headers", or maybe we want to be stronger with it and call it "enforced_headers" or "required_headers" even (and then maybe enforce they exist before producing, similar to the checks you're doing in the serializer directly...but you might be able to further generalize that to headers as a whole).

Reply:
TBH, nothing better than "extra" comes to my mind. 
These are the headers that will be set on top of the already specified by user, so word "extra" seems to fit.

- "Required" would mean that a user must specify the headers, which is not exactly the case
- "Enforced" also seems  to miss the target